README.md

```diff
@@ -110,8 +110,7 @@ GoLang: SDK|Every Thursday alternating between 10:00 and 15:00 PDT|[Google Doc](
 Instrumentation: Semantics|Every Monday at 11:30 PT|[Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)|[Zoom](https://zoom.us/j/8203130519?pwd=elFSMWxOSVN0Qjk3cVJ1dzZoSnpnZz09)|[Slack](https://cloud-native.slack.com/archives/C01QZFGMLQ7)
 Instrumentation: General|Every Tuesday at 16:00 PT|[Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)|[Zoom](https://zoom.us/j/6729396170?pwd=UFRIL0U0VzczUk1RK0hYaXJITXVldz09)|[Slack](https://cloud-native.slack.com/archives/C01QZFGMLQ7)
 Instrumentation: Messaging|Every Thursday at 8:00 PT|[Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)|[Zoom](https://zoom.us/j/8203130519?pwd=elFSMWxOSVN0Qjk3cVJ1dzZoSnpnZz09)|[Slack](https://cloud-native.slack.com/archives/C01QZFGMLQ7)
-Java: SDK|Every Friday at 08:00 PDT|[Google Doc](https://docs.google.com/document/d/1oHpisisco8J7Yk73ovx3eeXaokSMqgVuEuWNwPOajtY/edit)|[Zoom](https://zoom.us/j/8287234601)|[GitHub Discussions](https://github.com/open-telemetry/opentelemetry-java/discussions)|
-Java: Instrumentation|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/edit?usp=sharing)|[Zoom](https://zoom.us/j/8203130519)|[GitHub Discussions](https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions)|
+Java: SDK + Instrumentation|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/)|[Zoom](https://zoom.us/j/8203130519)|[SDK](https://cloud-native.slack.com/archives/C014L2KCTE3) and [Instrumentation](https://cloud-native.slack.com/archives/C014L2KCTE3) Slack channels|
```

Comment:
two slack links points to the same place

Reply:
thx! fixed